File: fb.c
#include <stdio.h>

int FB_LIMIT = 100;

int main() {
	for (int i = 1; i <= FB_LIMIT; i++) {
		int found = 0;

		if (i % 3 == 0) {
			found = 1;
			printf("Fizz");
		}

		if (i % 5 == 0) {
			found = 1;
			printf("Buzz");
		}

		if (!found) {
			printf("%d", i);
		}

		printf("\n");
	}

	return 0;
}

File: fb.go
package main

import "fmt"

const fbLimit = 100

func main() {
	for i := 1; i <= fbLimit; i++ {
		found := false

		if i % 3 == 0 {
			found = true
			fmt.Printf("Fizz")
		}

		if i % 5 == 0 {
			found = true
			fmt.Printf("Buzz")
		}

		if !found {
			fmt.Printf("%d", i)
		}

		fmt.Printf("\n")
	}
}
